packages/web/components/layouts/main.tsx

```diff
@@ -198,3 +191,62 @@ const OsmosisFullLogo: FunctionComponent<{
     }}
   />
 );
+
+const Drawer = ({
+  children,
+  showSidebar,
+  isMobile,
+  height,
+}: {
+  children: ReactNode;
+  showSidebar: boolean;
+  isMobile: boolean;
+  height: number;
+}) => {
+  const windowLoading = height <= 0;
+  if (windowLoading) {
+    return <div className="hidden">{children}</div>;
```

Comment:
Consider returning `null` here instead of a hidden div?

Reply:
@jonator I reverted `useWindowSize` to the way it was implemented before by using `windowLoading`.

also I am now returning `null` as you suggested :)